package.json

```diff
@@ -39,5 +39,12 @@
   "engines": {
     "node": ">=20.9.0"
   },
-  "packageManager": "pnpm@8.15.8"
+  "packageManager": "pnpm@8.15.8",
+  "pnpm": {
+    "overrides": {
+      "prismjs": "1.30.0",
```

Comment:
this one keeps popping up, how come a syntax highlighter is a dependency everywhere?

Reply:
it's a storybook dep, no harm bumping it anyway I think. Other ones are more serious, since they affect transpiled code under certain conditions